packages/main/src/tray-animate-icon.ts

```
@@ -84,21 +84,12 @@ export class AnimatedTray {
     } else {
       name = `-${iconName}`;
     }
-    let suffix: string;
-    // on Linux, always pickup dark icon
-    if (isLinux()) {
-      suffix = 'Dark';
-    } else if (isMac()) {
-      // on Mac, always pickup template icon
+    let suffix: string = '';
```

Biome (2.4.7)

[error] This type annotation is trivially inferred from its initialization.

(lint/style/noInferrableTypes)